mapreader/load/images.py

```diff
@@ -223,6 +225,18 @@ def imagesConstructor(
             if "image_path" not in self.images["parent"][parent_basename].keys():
                 self.images["parent"][parent_basename]["image_path"] = parent_path
 
+    @staticmethod
+    def _check_image_mode(image_path):
+        try:
+            img = Image.open(image_path)       
+        except:
```

Comment:
Don't use bare excepts. Explicitly catch the exceptions raised here](https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.open), or as a very minimum catch `Exception`.

Also, the relevant tests don't cover this error condition.

Reply:
Have updated to handle PIL.UnidentifiedImageError (which I think is the only one we can get other than FileNotFoundError since the other two are related to ``mode`` and ``format`` arguments which we don't specify).